sdk-api-src/content/memoryapi/nf-memoryapi-mapviewoffile3.md

```diff
@@ -75,7 +75,7 @@ If <i>BaseAddress</i> is not <b>NULL</b>, then any provided <a href="/windows/wi
 
 The offset from the beginning of the section.
 
-The offset must be 64k aligned.
+The offset must be 64k aligned or aligned to GetLargePageMinimum when MEM_LARGE_PAGES is used in AllocationType. Furthermore offset must be page alligned the the underlying page size granted by VirtualAlloc2 when MEM_REPLACE_PLACEHOLDER is used in AllocationType.
```

Comment:
Please correct the last sentence you added here. It's not possible to infer the intent with the typo. Thanks!

Reply:
Typo resolved. Its about this behaviour:
Normally: 64k offsets work
Large Pages: 64k will fail and a multiple of GetLargePageMinimum must be used.
 
Having a MEM_REPLACE_PLACEHOLDER only offsets work that correspond to the pagesize requested when the placeholder was created.